src/styles/components/_header.scss

```diff
@@ -1,7 +1,40 @@
+$breakpoint-tablet: 820px;
+
 .cmp-header {
   &__logo {
     max-width: 12rem;
     width: 100%;
     height: auto;
   }
 }
+
+.home {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1rem;
+}
+
+.home-top {
+  display: flex;
+  justify-content: space-between;
+
+  h1 {
+    font-size: 2.5rem;
+  }
+
+  &__top-description {
+    font-size: 1.1rem;
+  }
+
+  &__top-image {
+    width: 100%;
+  }
+}
+
+@media only screen and (min-width: $breakpoint-tablet) {
+  .home {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 1rem;
+  }
+}
```

Comment:
We're going to run into some conflicts here, since Terri changed this in a slightly different way. Would you mind matching this up as closely as you can with her [draft PR](https://github.com/sparkbox/accessible-components/pull/23/files#diff-05b077095f8f5f67dbe63f0781e1c882644b191edea604bc390e4dd0fc4c8bcf) to help mitigate those conflicts?